extract/extract/utils.py

```diff
@@ -9,8 +9,10 @@
 logger = getLogger(__name__)
 
 _certificate_category_names = {
-    "LAPTOP": ["LAPTOPS", "NOTEBOOKS"],
-    "SMARTPHONE": ["SMARTPHONES", "MOBILE_PHONES"],
+    "LAPTOP": ["LAPTOPS", "NOTEBOOKS", "LAPTOP", "NOTEBOOK"],
+    "SMARTPHONE": ["SMARTPHONES", "MOBILE_PHONES", "SMARTPHONE", "MOBILE_PHONE"],
+    "PRINTER": ["PRINTERS", "PRINTER"],
+    "TABLET": ["TABLETS", "TABLET"],
```

Comment:
Could you maybe update the keys by using the category values from the enum: https://github.com/calgo-lab/green-db/blob/302c6ebb27bcd387dbcf37004e4bde28114531d7/core/core/domain.py#L123

by using e.g. `ProductCategory.LAPTOP.value`

Reply:
Ah right, done now